test/unit_tests/js/modules/util/behavior-spec.js

```diff
@@ -128,28 +117,26 @@ describe( 'behavior', function() {
       expect( behaviorDom.length === 1 ).to.equal( true );
     } );
 
-    it( 'should throw an error when passed an invalid behavior selector',
-      () => {
-        const behaviorSelector = 'a[href^="#"]';
-        const errorMsg = '[data-js-hook*=behavior_' +
-                       behaviorSelector + '] not found in DOM!';
-        const findFunction = behavior.find.bind( this, behaviorSelector );
-        expect( findFunction ).to.throw( Error, errorMsg );
-      } );
+    it( 'should throw an error when passed an invalid behavior selector', () => {
+      const behaviorSelector = 'a[href^="#"]';
+      const errorMsg = '[data-js-hook*=behavior_' +
+                     behaviorSelector + '] not found in DOM!';
+      const findFunction = behavior.find.bind( this, behaviorSelector );
+      expect( findFunction ).to.throw( Error, errorMsg );
+    } );
   } );
 
   describe( 'remove function', () => {
-    it( 'should remove the event callback for the specific behavior hook',
-      () => {
-        const spy = sinon.spy();
-        const linkDom = document.querySelector( 'a[href^="#"]' );
-        behavior.attach( linkDom, 'click', spy );
-        triggerEvent( linkDom, 'click' );
-        expect( spy.called ).to.equal( true );
-        spy.reset();
-        behavior.remove( linkDom, 'click', spy );
-        triggerEvent( linkDom, 'click' );
-        expect( spy.called ).to.equal( false );
-      } );
+    it( 'should remove the event callback for the specific behavior hook', () => {
+      const spy = sinon.spy();
```

Comment:
Lets get rid of Sinon here and throughout the other tests.  We should just switch to `jest.spyOn`.  This should allow us to eliminate Sinon as a depencency.

Reply:
Yeah I agree, since this is passing in its current state I thought I'd leave that to another PR for review sanity. Chai can be removed too, though we have to change the [matcher syntax](https://facebook.github.io/jest/docs/en/expect.html).